lib/ex_unit/extra.ex

```diff
@@ -25,4 +25,38 @@ defmodule ExUnit.Extra do
     end
   end
 
+
+  @doc """
+  Compares two MapSets and asserts that they are equal.
+
+  Both arguments are passed through MapSet.new/1 before being compared.
+
+    iex> import ExUnit.Extra
+    ...> assert_set_equality [:a, :b], [:b, :a]
+    true
+
+  """
+  defmacro assert_set_equality(set_a, set_b) do
+    quote do
+      assert MapSet.new(unquote(set_a)) == MapSet.new(unquote(set_b))
+    end
+  end
+
+
+  @doc """
+  Compares two MapSets and asserts that they are NOT equal.
+
+  Both arguments are passed through MapSet.new/1 before being compared.
+
+    iex> import ExUnit.Extra
+    ...> refute_set_equality [:a, :b], [:a]
+    false
+
+  """
+  defmacro refute_set_equality(set_a, set_b) do
```

Comment:
Both of these would work just fine as functions rather than macros

Reply:
The macro let me get away with a few things (not importing ExUnit.Assertions, not adding to the call stack) but I agree it's of minimal value; I've moved them to functions.